tsconfig.frontend.json

```diff
@@ -8,5 +8,6 @@
     "inlineSources": true,
     "strictPropertyInitialization": false,
     "lib": ["dom", "es5"]
-  }
+  },
+  "exclude": ["app.js", "lib", "examples"]
```

Comment:
It started to check /lib and /examples folders when invoked on its own :thinking: